packages/frontend/src/hooks/useQueryParams.tsx

```diff
@@ -14,9 +14,7 @@ const useQueryParams = () => {
   const updateQueryParams = (updates: { [index: string]: string | undefined }) => {
     Object.assign(queryParams, updates)
 
-    history.push({
-      search: stringify(queryParams),
-    })
+    navigate(location.pathname + '?' + stringify(queryParams))
```

Comment:
suggestion: use string interpolation instead

```suggestion
    navigate(`${location.pathname}?${stringify(queryParams)}`)
```